src/newadagrams.js

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
I noticed this was your attempt at creating a class based solution! Great work trying to stretch yourself!